src/views/login/content/LoginCard.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <el-form
+    :model="loginForm"
+    :rules="rules"
+    ref="LoginCard"
+    label-width="120px"
+  >
+    <el-form-item label="Email" prop="email">
+      <el-input v-model="loginForm.email"> </el-input>
+    </el-form-item>
+    <el-form-item label="Password" prop="password">
+      <el-input v-model="loginForm.password"> </el-input>
+    </el-form-item>
+    <el-form-item>
+      <el-button id="loginButton" type="primary" @click="Login">
+        Login
+      </el-button>
+    </el-form-item>
+  </el-form>
+</template>
+
+<script>
+import { email } from "./validationRules.js";
+import { password } from "./validationRules.js";
+
+export default {
+  data() {
+    return {
+      loginForm: {
+        email: "",
+        password: "",
+      },
+      rules: {
+        email,
+        password,
+      },
+    };
+  },
+  methods: {
+    Login() {
+      console.log("login");
```

Comment:
you can also create some validation flow. As i remember, element-ui form provide you ability to invoke verify method, which additionaly will check corectness of data. You should have access to this method via form's ref like `this.$refs.LoginCard` in this case